k8s/demo/common/bar/bar-app.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   values:
     java:
       replicas: 2
-      image: hmctspublic.azurecr.io/bar/api:pr-502-5aca622-20210820152827
+      image: hmctspublic.azurecr.io/bar/api:pr-502-5aca622-20210820152827   #{"$imagepolicy": "flux-system:bar-app-bar-api"}
```

Comment:
@chrisorisawayi this should reference demo.yaml if it is not prod-* image

Reply:
now updated to exclude k8s/demo/* as discussed.